### PR TITLE
Various changes for better data migration (from Ralph2.0)

### PR DESCRIPTION
**Pull request by ghost**
- Extracted db creation to seperate file,
- added null=True to Services.
- added check if all rows from file was imported etc.

Connected to #1508

### CODE REVIEW

File: src/ralph/data_center/migrations/0002_auto_20150622_0957.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```
Comment (szok) [inline src/ralph/data_center/migrations/0002_auto_20150622_0957.py]: Is not it better to use one migration?
./reset.sh

Comment (mkurek) [inline src/ralph/data_center/migrations/0002_auto_20150622_0957.py]: run reset script again to merge it into 0001 migration

File: src/ralph/data_importer/management/commands/importer.py [hunk 2]
```diff
@@ -146,7 +147,11 @@ def from_file(self, options):
                             'row data: {}'.format(zip(headers, dataset[idx])),
                             '',
                         ])
-                        logger.error(error_msg)
```
Comment (szok) [inline src/ralph/data_importer/management/commands/importer.py:149]: We must login error, we will not know what was.

File: src/ralph/data_center/migrations/0002_auto_20150622_0957.py (hunk 1 above)
Comment (ghost) [inline src/ralph/data_center/migrations/0002_auto_20150622_0957.py]: This is already the ouput from`scripts/reset.sh`, but i can run it again ;)

File: src/ralph/data_importer/management/commands/importer.py (hunk 2 above)
Comment (ghost) [inline src/ralph/data_importer/management/commands/importer.py:149]: my bad.., I'll readd it.

File: src/ralph/assets/models/assets.py [hunk 3]
```diff
@@ -352,3 +352,8 @@ def clean(self):
                     'barcode': error_message
                 }
             )
+
+    def save(self, *args, **kwargs):
+        self.barcode = self.barcode or None
```
Comment (mkurek) [inline src/ralph/assets/models/assets.py]: please add some comment why we're doing such a thing

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

File: docs/data_migration.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+## CLI import
+
+Example command which imports data could look like this::
+
+    $ ralph importer --type file ./path/to/DataCenterAsset.csv --model_name DataCenterAsset
+
+or
+
+    $ ralph importer --type zip ./path/to/exported-files.zip
```
Comment (szok) [inline docs/data_migration.md]: add --skipid

**Comment by szok** [comment]
:ok:

**Pull request merged**